src/DDoSX/HstsClient.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\Hsts;
+use UKFast\SDK\DDoSX\Entities\HstsRule;
+
+class HstsClient extends BaseClient
+{
+    /**
+     * @var string $basePath
+     */
+    protected $basePath = 'ddosx/';
+
+    const HSTS_MAP = [];
+    const HSTS_RULE_MAP = [
+        'max_age' => 'maxAge',
+        'include_subdomains' => 'includeSubdomains',
+        'record_name' => 'recordName'
+    ];
+
+    /**
+     * Get the HSTS enabled status
+     * @param $domainName
+     * @return Hsts
+     */
+    public function getStatus($domainName)
+    {
+        $response = $this->get('v1/domains/' . $domainName . '/hsts');
+        $body = $this->decodeJson($response->getBody()->getContents());
+
+        return $this->serializeHsts($body->data);
+    }
+
+    /**
+     * Get a paginated list of HSTS rules for a domain
+     * @param $domainName
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|\UKFast\SDK\Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getRules($domainName, $page = 1, $perPage = 15, $filters = [])
```

Comment:
DDoSX uses 20 as a default:

```suggestion
    public function getRules($domainName, $page = 1, $perPage = 20, $filters = [])
```